docs/LICENSE_OF_DEPENDENCIES.md

```diff
@@ -72,6 +72,7 @@ following works:
 - github.com/mailru/easyjson [MIT License](https://github.com/mailru/easyjson/blob/master/LICENSE)
 - github.com/matttproud/golang_protobuf_extensions [Apache License 2.0](https://github.com/matttproud/golang_protobuf_extensions/blob/master/LICENSE)
 - github.com/mdlayher/apcupsd [MIT License](https://github.com/mdlayher/apcupsd/blob/master/LICENSE.md)
+- github.com/mdlayher/wireguardctrl [MIT License](https://github.com/mdlayher/wireguardctrl/blob/master/LICENSE.md)
```

Comment:
Need to add licenses for all new repos, this doesn't necessarily need to be done in this PR, we can do it at merge time.

Reply:
Got it, I've unversioned changes to this file so that it can be done at merge time.